speakeasy/binemu.py

```diff
@@ -703,7 +703,7 @@ def read_mem_string(self, address, width=1, max_chars=0):
         i = 0
 
         if width == 1:
-            decode = 'latin1'
+            decode = 'utf-8'
```

Comment:
can you explain this change?

Reply:
this change tries to align the ways how speakeasy reads and writes strings from/to memory. I felt that using utf-8 in either case is more error-resistant than using latin1 in one code path and utf-8 in the other.